migrations/000001_create_monitor_table.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS monitors (
```

Comment:
For now, we just have the monitors table. 
After finish the CRUD operation for this entity I will start to develop the other ones

Reply:
I was thinking if instead of use a SQL database a NOSql will fit well in our case. WDYT?